packages/radio/README.md

```diff
@@ -1,4 +1,39 @@
-Base Package
+# Radio
 
-Don't run any npm script in here. It's just a basic folder to enable us create
-new packages faster.
+Radio component is used in forms when a user needs to select a single value from
+several options.
+
+## Installation
+
+```sh
+yarn add @chakra-ui/radio
+
+# or
+
+npm i @chakra-ui/radio
+```
+
+## Import component
+
+```jsx
+import { Radio } from "@chakra-ui/radio"
+```
+
+## Usage
+
+```jsx
+<Radio>This is a radio</Radio>
+```
+
+### RadioGroup
+
+RadioGroup is used to bind multiple radios into a group, and it indicates
+whether which option is selected.
+
+```jsx
+<RadioGroup defaultValue={["one", "two"]}>
```

Comment:
RadioGroup only takes a single value not an array, since only one radio can be checked at a time. Please change this.

Reply:
Thanks for spotting this @segunadebayo. It has been resolved.